solution/ui/regulations/eregs-vite/src/main.js

```diff
@@ -31,6 +31,11 @@ router.beforeEach((to) => {
         if (_isArray(value) && key != "type") {
             to.query[key] = value[0];
         }
+
+        // Encode the `q` query param to handle special characters in API calls etc
+        if (key == "q") {
+            to.query[key] = encodeURIComponent(value);
```

Comment:
this keeps the URL clean and encoded as normal, i.e. spaces are still `+` marks.  This will help existing links to still work as expected.